scanner/indexer/indexer_test.go

```diff
@@ -0,0 +1,54 @@
+package indexer
+
+import (
+	"testing"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_parseImageURL(t *testing.T) {
+	tests := []struct {
+		name    string
+		arg     string
+		want    name.Reference
+		wantErr string
+	}{
+		{
+			name:    "empty URL",
+			arg:     "",
+			wantErr: "could not parse ref",
+		},
+		{
+			name:    "no schema",
+			arg:     "foobar",
+			wantErr: "could not parse ref",
```

Comment:
interesting, it does not default to `https` or anything? I think images usually default to `https`

Reply:
You mean the final ref? It doesn't allow a single name because strict won't add a default host to the image reference.

Reply:
I added explicit checks and updated the error message, although redundant, they don't harm either.